File: erlang/two-fer/src/two_fer.erl
-module(two_fer).

-export([two_fer/0, two_fer/1]).

-spec two_fer() -> Result :: string().
two_fer() ->
    two_fer("you").

-spec two_fer(Name :: string()) -> Result :: string().
two_fer(Name) ->
    io_lib:format("One for ~s, one for me.", [Name]).

File: go/two-fer/two_fer.go
// Package twofer implements the Two Fer exercise.
package twofer

// Given a name, return a string with the message, "One for name, one for me."
func ShareWith(name string) string {
	if name == "" {
		return ShareWith("you")
	}

	return "One for " + name + ", one for me."
}
